Stylesheets/Components/ratings.css

```diff
@@ -0,0 +1,40 @@
+.rating-container {
+    width: fit-content;
+    height: fit-content;
+    gap: 1rem;
+    background-color: var(--color-background-dull);
+    padding: 8px;
+    border-radius: var(--container-border);
+}
+
+.rating-star{
+    margin:0;
+    position:relative;
+}
+
+.rating-container.badge {
+    position: static;
```

Comment:
Position will be static default, any reason for specifying it. Everything else looks good

Reply:
I am using badge component class, and in badge.css I had given position as absolute in badge.css